### DISCUSSION TITLE
如何将配置文件加密，以便在公共电脑上使用

### DISCUSSION THREAD

**Discussion by fffffffflyy** [v2fly/v2ray-core#2660] [General]
由于需要在公共电脑上使用v2ray，以明文形式存在的配置文件会泄漏服务器信息，这是不安全的。如何解决这个问题？
使用环境为windows

**Comment by AkinoKaede** [comment]
You can try to use it with other software like [age](https://github.com/FiloSottile/age)

**Reply by fffffffflyy** [reply]
请问v2ray怎么读取age加密后的配置文件？

**Reply by nursery01** [reply]
先有雞還是先有蛋

你啟動完直接把json刪除了不就行了
